lib/web_ui/lib/src/engine/pointer_binding/event_position_helper.dart

```diff
@@ -33,6 +43,24 @@ ui.Offset computeEventOffsetToTarget(DomMouseEvent event, DomElement actualTarge
   return ui.Offset(event.offsetX, event.offsetY);
 }
 
+/// Computes the offsets for input nodes, which live outside of the shadowDOM.
+/// Since inputs can be transformed (scaled, translated, etc), we can't rely on
+/// `event.offset` to be accurate. `_computeOffsetRelativeToActualTarget` only
```

Comment:
> Since inputs can be transformed (scaled, translated, etc), we can't rely on `event.offset` to be accurate.

You are rightfully stating the reason the previous implementation didn't work. But that's not the reason we can't rely on `event.offset`. The reason we can't use `event.offset` is because it is relative to the input itself and not to the flutter-view.

Reply:
Updated